.github/workflows/release.yml

```diff
@@ -1,9 +1,7 @@
 name: Release Charts
 
 on:
-  push:
-    branches:
-      - main
+  workflow_dispatch:
```

Comment:
Are we intending to only manually release the chart?

Reply:
This should just add the option to release using a manual trigger button. So, I think it would be both. Not sure if I put this in there or Amndeep, but if I did, I do not recall the exact reason but would assume I needed to manually release for some reason. Again, just my assumption.

Reply:
Talked about this on slack/voice.

It looks like the release action names the release after the latest tag.  Ergo you need to have a tagged commit ready for it.  Usually if you’re cutting releases from master then the tag should originate there instead of on a branch that then has to go through at least a merge commit.  Therefore you need time to do it before you manually trigger the process.  In heimdall and the saf cli we handle this by having a draft release action and then the release process happens when we manually tell it to release.  This hem action doesn’t allow for draft releases so we gotta trigger it manually.